src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/BaseOptions.java

```diff
@@ -293,13 +293,13 @@
 
     /**
      * Argument for command-line option {@code -tagletpath}.
-     * The path to Taglets
+     * The path to custom taglets.
```

Comment:
Less important (since it's an internal comment) but see suggestion for help string.